pkg/kubernetes/kubernetes.go

```diff
@@ -375,7 +374,13 @@ func (k *Kubernetes) applyCSVs(ctx context.Context, resources []unstructured.Uns
 }
 
 // InstallPerconaCatalog installs percona catalog and ensures that packages are available.
-func (k *Kubernetes) InstallPerconaCatalog(ctx context.Context) error {
+func (k *Kubernetes) InstallPerconaCatalog(ctx context.Context, version *goversion.Version) error {
+	if version == nil {
+		// No need to check the version if not provided.
+		k.l.Debug("No version provided to InstallPerconaCatalog")
+		return nil
```

Comment:
Shouldn't this be considered an error?

Reply:
Same as https://github.com/percona/everest/pull/67#discussion_r1525047315